packages/icons/scripts/build.js

```diff
@@ -28,42 +28,70 @@ const getFilePaths = dir => readdirSync(dir).map(file => `${dir}/${file}`);
 const writeSprite = (srcDir, spritePath, idPrefix) => {
   console.info(`\tgenerating sprite at:\n\t${spritePath}\n`);
 
-  writeFileSync(
-    spritePath,
-    getFilePaths(srcDir).reduce((sprites, file) => {
-      return sprites.add(
-        `${idPrefix}-${path.basename(file, ".svg")}`,
-        readFileSync(file, "utf8")
-      );
-    }, svgstore({ renameDefs: true }))
-  );
+  const store = svgstore({ renameDefs: true });
+
+  getFilePaths(srcDir).forEach(file => {
+    const iconName = `${idPrefix}-${path.basename(file, ".svg")}`;
+    const content = readFileSync(file, "utf8");
+    const optimized = optimize(content, {
+      plugins: [
+        { name: "removeDoctype", active: true },
+        { name: "removeComments", active: true },
+        { name: "cleanupAttrs", active: true },
+        { name: "mergeStyles", active: true },
+        { name: "inlineStyles", active: true },
+        { name: "minifyStyles", active: true },
+        { name: "convertStyleToAttrs", active: true },
+        { name: "removeStyleElement", active: true },
+        { name: "removeScriptElement", active: true },
+        { name: "removeTitle", active: true },
+        { name: "removeDesc", active: true },
+        { name: "removeDimensions", active: true },
+        { name: "removeUselessDefs", active: true },
+        { name: "removeUselessStrokeAndFill", active: true },
+        { name: "removeViewBox", active: false },
+        { name: "removeEmptyContainers", active: true }
+      ]
+    }).data;
+    store.add(iconName, optimized);
+  });
+
+  writeFileSync(spritePath, store);
 };
 
 const optimizeWithSVGO = spritePath => {
   console.info("\toptimizing sprite\n");
 
-  readFile(spritePath, "utf8", (err, data) => {
-    if (err) {
-      throw err;
-    }
+  try {
+    const data = readFileSync(spritePath, "utf8");
 
     const result = optimize(data, {
       path: spritePath,
       multipass: true,
       plugins: [
-        {
-          name: "removeDoctype",
-          active: true
-        },
-        {
-          name: "cleanupIDs",
-          active: false
-        }
+        { name: "removeDoctype", active: true },
+        { name: "removeComments", active: true },
+        { name: "cleanupAttrs", active: true },
+        { name: "mergeStyles", active: true },
+        { name: "inlineStyles", active: true },
+        { name: "minifyStyles", active: true },
+        { name: "convertStyleToAttrs", active: true },
+        { name: "removeStyleElement", active: true },
+        { name: "removeScriptElement", active: true },
+        { name: "removeTitle", active: true },
+        { name: "removeDesc", active: true },
+        { name: "removeDimensions", active: true },
+        { name: "removeUselessDefs", active: true },
+        { name: "removeUselessStrokeAndFill", active: true },
+        { name: "removeViewBox", active: false },
+        { name: "removeEmptyContainers", active: true }
```

Comment:
It would be cool to have one constant for these plugins we can use here.
Will be easier the next time we need to touch these.

Reply:
good thought, I'll update that.